### ISSUE TITLE
Setup fails when overwriting other fork. Any way to fix?

### ISSUE THREAD

**Issue by Din-BH** [jordanruthe/homeassistant-phyn#20]
Hi. I tried to shift from the rsocko fork to this fork to get the latest features, but I get the following error after hot-replacing. Understandable in one way, but is there any way to fix this without starting from scratch. Tried starting from scratch as well, but that also did not work... My device is a Phyn.

```
Logger: homeassistant.config_entries
Source: config_entries.py:406
First occurred: 21:21:28 (2 occurrences)
Last logged: 21:22:35

Error setting up entry V10 for phyn
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/phyn/__init__.py", line 32, in async_setup_entry
    phyn_brand=entry.data["Brand"].lower(), session=session,
               ~~~~~~~~~~^^^^^^^^^
KeyError: 'Brand'

```

**Comment by jordanruthe** [comment]
v2024.1.4 should fix this (commit bb99fcb ).

**Comment by Din-BH** [comment]
Thanks! That did the trick.
